e2e-tests/path-prefix/gatsby-config.js

```diff
@@ -58,6 +58,29 @@ module.exports = {
             title: `assetPrefix + pathPrefix RSS Feed`,
             output: `rss.xml`,
           },
+          {
+            query: `
+              {
+                pages: allSitePage {
+                  nodes {
+                    path
+                  }
+                }
+              }
+            `,
+            serialize({ query: { site, pages } }) {
+              return pages.nodes.map(node => {
+                return {
+                  description: `A sample page hello world suh dude`,
+                  date: `10-08-1990`,
+                  url: `${site.siteMetadata.siteUrl}${pathPrefix}${node.path}`,
+                }
+              })
+            },
+            title: `feedUrl RSS Feed`,
+            feedUrl: `http://localhost:9000/rss-2.xml`,
```

Comment:
So in this case, output isn't used whatsoever?

Reply:
`output` is still used for saving the file to disk (`gatsby-node.onPostBuild`)

Reply:
For clarity/conciseness then, it'd probably be a good idea to not include the output file in feedUrl, and infer it from the existing output?

Reply:
that's fits my usecase.
currently my feedUrl is `https://example.org/${output}`.

I left intentionally as a escape-hatch for even more complex use-cases, where an user can opt-out of the magic.

I can rewrite this portion if needed (: (but I think we should change it to something like `baseUrl` as it won't be the final one in the DOM)

Reply:
I think `siteUrl` is typically used for this--right? i.e. `siteMetadata.siteUrl` refers to the deployed URL of your application.

Reply:
(not suggesting you change the name--just an observation!)